.idea/.gitignore

```diff
@@ -0,0 +1,6 @@
+# Default ignored files
```

Comment:
I suggest adding `.idea` to your `.gitignore` file or the `.gitignore_global` file.